domain/port/state/watcher_test.go

```diff
@@ -110,38 +111,30 @@ FROM unit_endpoint
 *   - https is open on endpoint 2 on unit 2
  */
 
-func (s *watcherSuite) TestGetMachinesForUnitEndpoints(c *gc.C) {
+func (s *watcherSuite) TestGetMachinesForEndpoints(c *gc.C) {
```

Comment:
**todo:** the actually test `GetMachineNamesForUnitEndpoints`, please fix the name.